hack/tools/runtime-openapi-gen/main.go

```diff
@@ -59,19 +53,25 @@ func main() {
 	}
 
 	c := catalog.New()
-	_ = runtimehooksv1.AddToCatalog(c)
-	_ = v1alpha2.AddToCatalog(c)
-	_ = v1alpha3.AddToCatalog(c)
+	if err := runtimehooksv1.AddToCatalog(c); err != nil {
```

Comment:
You can ignore the errors as before like we always do when we add a package to a scheme. We followed the pattern from scheme and are just panicking if something goes wrong

Reply:
Actually the only error which can occur is actually from the upstream scheme, so I would ignore it just like we do usually with that one